composer.json

```diff
@@ -13,7 +13,9 @@
         "silverstripe/mimevalidator": "2.3.x-dev"
     },
     "require-dev": {
```

Comment:
travis config also includes `silverstripe/admin` and `silverstripe-themes/simple` - do we need those here?

Reply:
I don't think so?  We're not running behat here so I don't see why we would.